core/shared/src/main/scala/scalaz/zio/Clock.scala

```diff
@@ -16,7 +16,7 @@ trait Clock {
 object Clock {
 
   object Live extends Clock {
-    def currentTime(unit: TimeUnit): IO[Nothing, Long] = IO.sync(unit.convert(System.nanoTime(), NANOSECONDS))
+    def currentTime(unit: TimeUnit): IO[Nothing, Long] = IO.sync(unit.convert(System.currentTimeMillis(), NANOSECONDS))
```

Comment:
hmm,  currentTimeMillis return milliseconds, but in second argument of convert is NANOSECONDS.
Is this a typo ?